test/core/templates/module-test.1.js

```diff
@@ -0,0 +1,33 @@
+const test = require('tape'),
```

Comment:
I just feel like this whole file maybe could be merged into `test/core/sequencer/benchmark.js` here: https://github.com/publiclab/image-sequencer/pull/1093/files#diff-57fdbe0dfda9b326e4947de8d679f8b6R15, and this would all be a bit easier to see side-by-side, you know? What do you think?

Reply:
Actually this is something not feasible as wasm tests can not be browserifed due to some imitaions of browserify.

Reply:
But is this test file browserified? benchmark.js? I didn't think so...

Reply:
```js
 "test": "TEST=true istanbul cover tape test/core/*.js test/core/ui/user-interface.js test/core/modules/*.js | tap-spec; browserify test/core/sequencer/image-sequencer.js test/core/sequencer/chain.js test/core/sequencer/meta-modules.js test/core/sequencer/replace.js test/core/sequencer/import-export.js test/core/sequencer/run.js test/core/sequencer/dynamic-imports.js test/core/util/*.js test/core/sequencer/benchmark.js | tape-run --render=\"tap-spec\"",

```

The bechmarks.js is a browserified file as you can have a look here.

Reply:
That's right! But is there any reason we /have/ to run benchmarks in the browser environment? Presumably without `wasm` the computational intensity would be comparable whether in a browser or not. 

I tried running `node test/core/sequencer/benchmarks.js` and it ran in the node environment successfully -- although I hit a memory limit that we may need to overcome by adding this to the benchmarks.js file:

`require('events').EventEmitter.prototype.setMaxListeners(1000);`

(like in this commit: https://github.com/publiclab/image-sequencer-app/commit/1256685ebe4e81e70cad5ad9ba1ffc70a8ebb69d#diff-e6a5b42b2f7a26c840607370aed5301aR1)

https://github.com/publiclab/image-sequencer/blob/b34ec00efdd89352191fe8016c3b364f7ef64aa5/package.json#L8

Reply:
So I think we can change that line to (is this right?) and then have a switch that makes the benchmarks run together instead of having all `wasm` benchmarks separately -- what do you think?:

```
"test": "TEST=true istanbul cover tape test/core/*.js test/core/ui/user-interface.js test/core/modules/*.js | tap-spec; node test/core/sequencer/benchmark.js; browserify test/core/sequencer/image-sequencer.js test/core/sequencer/chain.js test/core/sequencer/meta-modules.js test/core/sequencer/replace.js test/core/sequencer/import-export.js test/core/sequencer/run.js test/core/sequencer/dynamic-imports.js test/core/util/*.js | tape-run --render=\"tap-spec\"",
```